qt/scientific_interfaces/test/ISISReflectometry/Batch/BatchPresenterTest.h

```diff
@@ -118,6 +118,46 @@ class BatchPresenterTest : public CxxTest::TestSuite {
     verifyAndClear();
   }
 
+  void testJobRunnerGetProcessAll() {
+    auto presenter = makePresenter();
+    // need to check m_jobRunner->m_processAll somehow
+    EXPECT_EQ(m_jobRunner->getProcessAll(), false);
+    expectReductionResumed();
+    presenter->notifyResumeReductionRequested();
+    verifyAndClear();
+  }
+
+  void testJobRunnerGetProcessPartial() {
+    auto presenter = makePresenter();
+    // need to check m_jobRunner->m_processPartial somehow
+    EXPECT_EQ(m_jobRunner->getProcessPartial(), false);
+    expectReductionResumed();
+    presenter->notifyResumeReductionRequested();
+    verifyAndClear();
+  }
+
+  void testWarnProcessAllWhenReductionResumed() {
```

Comment:
Is it worth adding the inverse for these tests as well, i.e. that the warning is not displayed when the option is disabled?